src/compiler/crystal/command/playground.cr

```diff
@@ -34,6 +34,26 @@ class Crystal::Command
       end
     end
 
+    playground_url = "http://#{server.host || "localhost"}:#{server.port || "8080"}"
+
+    open_browser(playground_url)
+
     server.start
   end
+
+  # Open up the default browser automatically to the Playground.
+  private def open_browser(playground_url)
+    {% if flag?(:darwin) %}
+      system "open #{playground_url}"
+    {% elsif flag?(:win32) %}
+      system "start #{playground_url}"
+    {% elsif flag?(:linux) %}
+      system "xdg-open #{playground_url}"
+    # TODO
+    # {% elsif flag?(:freebsd) %}
```

Comment:
If you plan to uncomment these macro branches, I think `:freebsd` and `:openbsd` are included in `:linux` so you probably don't need them.

Reply:
Perfect, thanks @r00ster91!

Reply:
What? Why should BSDs match the Linux flag?

Reply:
Oh, then  probably `:unix` which will definitely include them.

Reply:
So, should we change `flag?(:linux)` to `flag?(:unix)`?

Reply:
Yes.

Reply:
Replaced `linux` with `unix` in 987e539a2ed645f68a874a96812275fbdd9bc589.